modules/migration-compatibility-guidelines-3-to-4.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/installing-3-4.adoc
+// * migrating_from_ocp_3_to_4/installing-restricted-3-4.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="migration-compatibility-guidelines-3-to-4_{context}"]
+= Compatibility guidelines
+
+You must install the {mtc-full} ({mtc-short}) Operator that is compatible with your {product-title} version.
+
+.Definitions
+
+control cluster:: The cluster that runs the {mtc-short} controller and GUI.
+remote cluster:: A source or destination cluster for a migration that runs Velero. The Control Cluster communicates with Remote clusters via the Velero API to drive migrations.
```

Comment:
VALE - RedHat.TermsSuggestions - Depending on the context, consider using 'through', 'by', 'from', 'on', or 'by using' rather than 'via'.

```suggestion
remote cluster:: A source or destination cluster for a migration that runs Velero. The Control Cluster communicates with Remote clusters from the Velero API to drive migrations.
```

Reply:
Went with "using" -- thanks for making me reconsider.